phpBB/phpbb/notification/method/webpush.php

```diff
@@ -93,7 +93,7 @@ public function get_type(): string
 	*/
 	public function is_available(type_interface $notification_type = null): bool
 	{
-		return parent::is_available($notification_type) && $this->config['webpush_enable']
```

Comment:
Honestly this could prob just be:

```php
return $this->config['webpush_enable']
    && $this->config['webpush_vapid_public']
    && $this->config['webpush_vapid_private'];
```